### PR TITLE
Edit Stream Schema

### PR DESCRIPTION
**Pull request by Smarker**
* Fixes any incorrect pipelinekeys, streamfactories
* Only shows one stream factory - facebook and instagram have two

### CODE REVIEW

File: project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js [hunk 1]
```diff
@@ -260,77 +262,70 @@ const defaultStreamMap = {
...
-  },
-  FacebookPost: {
-    pipelineKey: 'FacebookPost',
-    pipelineLabel: 'FacebookPost',
-    pipelineIcon: 'Facebook Post Icon',
-    streamFactory: 'FacebookPost',
+  Facebook: {
+    pipelineKey: 'Facebook',
+    pipelineLabel: 'Facebook',
+    pipelineIcon: 'Facebook Icon',
```
Comment (c-w) [inline project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js]: Shouldn't these be [font-awesome](https://fontawesome.com/v4.7.0/icons/) class-names? Like `fa fa-facebook-official` or `fa fa-twitter` (the latter we use in some of the seed data).

**Review by c-w — CHANGES_REQUESTED** [review]

File: project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js (hunk 1 above)
Comment (Smarker) [reply project-fortis-interfaces/src/components/Admin/Streams/StreamConstants.js]: I'll look into that

**Comment by kevinhartman** [comment]
I was only able to take a quick look here, but I'd like to note that the concept of a Stream Factory should not be exposed to the user (not sure if it is in this PR, but I'm mentioning it just in case). The correct factory should be chosen based on the settings provided by the user. In some cases, this may be implemented with a drop-down to select an identifier which is one-to-one mapped with an underlying stream factory, but this should not be strictly true.

**Comment by Smarker** [comment]
yes I agree, I didn't label it as stream factory, but it's good to note. from your comment, when you said settings provided by the user I thought of an idea. Rather than having them enter in the factory, we can just interpret the factory based on the fields they enter. (that way there's no need to have another select option the user needs to fill out)

e.g. for Instagram:
show both tags and location -> if they fill out both then create 2 records (one for instagramTag factory and one for instagramLocation factory)

**Comment by kevinhartman** [comment]
Perfect!

**Review by c-w — APPROVED** [review]

**Pull request merged**